app/lib/widgets/ToDoTaskItem.dart

```diff
@@ -72,227 +73,350 @@ class _ToDoTaskItemState extends State<ToDoTaskItem> {
-                      padding: const EdgeInsets.only(left: 8, top: 3),
-                      child: Text(
-                        widget.title,
-                        style: ToDoTheme.taskTitleTextStyle.copyWith(
-                          decoration: widget.isCompleted
-                              ? TextDecoration.lineThrough
-                              : null,
+                  Padding(
+                    padding: const EdgeInsets.only(
+                      right: 8,
+                    ),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.end,
+                      children: [
+                        const Padding(
+                          padding: EdgeInsets.all(8.0),
+                          child: Icon(
+                            FlutterIcons.check_circle_outline_mco,
+                            color: AppCommonTheme.primaryColor,
+                            size: 16,
+                          ),
                         ),
-                        overflow: TextOverflow.ellipsis,
-                      ),
+                        Text(
+                          widget.dateTime,
+                          style: ToDoTheme.completeCalendarTextStyle,
+                        ),
+                      ],
                     ),
                   ),
-                  // large screens
-                  if (deviceWidth > 600) ...[
-                    Padding(
-                      padding: const EdgeInsets.only(left: 40, right: 8),
-                      child: SvgPicture.asset(
-                        'assets/images/calendar-2.svg',
-                        color: isAllDay
-                            ? ToDoTheme.todayCalendarColor
-                            : ToDoTheme.calendarColor,
-                      ),
-                    ),
-                    Text(
-                      widget.dateTime,
-                      style: isAllDay
-                          ? ToDoTheme.todayCalendarTextStyle
-                          : ToDoTheme.calendarTextStyle,
-                    ),
-                    const SizedBox(
-                      width: 40,
-                    ),
-                    widget.hasMessage
-                        ? Row(
-                            children: [
-                              SvgPicture.asset(
-                                'assets/images/message.svg',
-                                height: 12,
-                                width: 12,
-                              ),
-                              Padding(
-                                padding: const EdgeInsets.only(left: 5),
-                                child: Text(
-                                  '$messageCount',
-                                  style: ToDoTheme.buttonTextStyle.copyWith(
-                                    color: ToDoTheme.primaryTextColor,
+                ],
+              ),
+            )
+          : Card(
```

Comment:
Long UI widget. Consider seperate widget for it and call here to improve ```build()``` readability.